rules/cloud/azure/azure_aad_secops _signin_failure_bad_password_threshold.yml

```diff
@@ -0,0 +1,27 @@
+title: Sign-in failure Bad Password Threshold
```

Comment:
`title: Sign-in Failure Bad Password Threshold` 

Rule rules\test\azure_aad_secops _signin_failure_bad_password_threshold.yml has a title that has not title capitalization. Words: 'failure'